x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -78,14 +77,15 @@ export function TimeseriesChart({
   const history = useHistory();
   const { annotations } = useAnnotationsContext();
   const { setPointerEvent, chartRef } = useChartPointerEventContext();
-  const { urlParams } = useUrlParams();
   const theme = useTheme();
   const chartTheme = useChartTheme();
 
-  const { start, end } = urlParams;
+  const xValues = timeseries
+    .map(({ data }) => data.map(({ x }) => x))
+    .flatMap((_) => _);
```

Comment:
Isn't this the same?
```suggestion
  const xValues = timeseries
    .flatMap(({ data }) => data.map(({ x }) => x))
```